app/src/main/java/com/example/unifood/recyclerview/RestaurantRecyclerView.kt

```diff
@@ -41,17 +43,23 @@ class RestaurantListAdapter(private var items: MutableList<Restaurant>) :
 
     inner class RestaurantViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         fun bind(item: Restaurant) {
-            Picasso.get().load(item.imageUrl)
-                .into(itemView.findViewById<ImageButton>(R.id.iconRestaurant_1))
+            val imageButton = itemView.findViewById<ImageButton>(R.id.iconRestaurant_1)
```

Comment:
`java.lang.ClassCastException: androidx.appcompat.widget.AppCompatImageView cannot be cast to android.widget.ImageButton
                                                                                                    	at com.example.unifood.recyclerview.RestaurantListAdapter$RestaurantViewHolder.bind(RestaurantRecyclerView.kt:46)`

You are trying to cast an ImageView to an ImageButton

Reply:
Because of it the whole view does not appear